README.md

```diff
@@ -89,19 +121,49 @@ Clone this repository to your desired folder:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
+
+
 > Describe 1 - 3 features you will add to the project.
 
+
 - [ ] **[new_feature_1]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
 
```

Comment:
- [ ] [Optional] Please remove this phrase `> Describe 1 - 3 features you will add to the project.` and replace these words `[new_feature_1]` with the features the project will have as this Highlight the most important and distinctive features of your project without overwhelming the reader. :smile:

Reply:
Hi @Alusp thanks for your reviews, I made the changes needed on the README